src/components/Hero/MdxHero/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { Heading, Stack } from "@chakra-ui/react"
+import * as React from "react"
+import Breadcrumbs, { IProps as BreadcrumbsProps } from "../../Breadcrumbs"
+
+interface MdxHeroProps {
+  breadcrumbs: BreadcrumbsProps
+  headingTitle: string
```

Comment:
To be consistent with the rest
```suggestion
  title: string
```